android/assets/jsons/translations/French.properties

```diff
@@ -1964,46 +1958,35 @@ Greetings, stranger. I am Hiawatha, speaker for the Iroquois. We seek peace with
 Does this trade work for you, my friend? = Cette offre vous convient-elle, mon ami ?
 Onoondaga = Onondaga
 Osininka = Osininka
-Grand River = Rivière Grand
+Grand River = Grande rivière
 Akwesasme = Akwesasne
- # Requires translation!
-Buffalo Creek = 
+Buffalo Creek = Buffalo Creek
 Brantford = Brantford
 Montreal = Montréal
 Genesse River = Rivière Genesse
 Canandaigua Lake = Lac Canandaigua
 Lake Simcoe = Lac Simcoe
 Salamanca = Salamanque
- # Requires translation!
```

Comment:
While looking at these, I couldn't find any official translation, so I guess you just reused the english version. I'm not sure whether we should do it, because this is already what happens by default.

I'm not sure what is the right way to treat this, maybe @yairm210 can provide some guidance here?

Reply:
Indeed, on that topic I did that especially to get rid of the "87% of translation done" only because some cities names only exists in original version. What do you guys think ?

Reply:
There's no need to reverse that change I think. Let's leave it as is and see if we get a response or if a guideline gets written in the future :)